src/api.js

```diff
@@ -0,0 +1,25 @@
+import axios from "axios";
+
+const api = axios.create({
+    baseURL: 'https://nc-news-m199.onrender.com/api'
+})
+
+// Articles
+
+const getArticles = () => {
+    return api.get('/articles').then(({ data }) => {
+        return data.articles
+    })
+}
+
+const getArticleById = (article_id) => {
+    return api.get(`/articles/${article_id}`).then(({ data }) => {
+        return data.article
+    })
+}
+const getUsers = () => {
```

Comment:
These bottom two calls aren't relevant to this ticket. Moving forward, please make sure that PRs only include code relating to the ticket they're for - this will be greatly appreciated by your future colleagues/ they'll tut at you if you don't.